README.md

```diff
@@ -67,3 +67,94 @@ and if desired for UI testing:
 ```bash
 $ ./multinode-demo/client.sh --tx_count 40 --threads 2 -z 400
 ```
+
+## High Performance Use Cases
+
+### Redis via Unix Domain Socket
+
+Redis is known as a very fast in-memory data structure server. To keep up with Solana
+speeds, it may be useful to enable Unix Domain Socket communication for added performance
+(in our unscientific testing, it gives about 10-40% or more depending on the operations).
```

Comment:
How about just "(about 10-40%, depending on the operation, see benchmarks below)"

Reply:
🆗